### ISSUE TITLE
qwen_image_train_network.py: error: unrecognized arguments

### ISSUE THREAD

**Issue by BenDes21** [kohya-ss/musubi-tuner#499]
Hi, Im trying to train a qwen edit lora, seems the args from the documentation :   

--control_image_path 
--prompt
 --resize_control_to_official_size

are not recognized 

my current settings  :   

accelerate launch \
  --num_cpu_threads_per_process 1 --mixed_precision bf16 src/musubi_tuner/qwen_image_train_network.py \
  --persistent_data_loader_workers \
  --network_module=networks.lora_qwen_image \
  --dit /workspace/models/diffusion_models/split_files/diffusion_models/qwen_image_bf16.safetensors \
  --vae /workspace/models/vae/split_files/vae/qwen_image_vae.safetensors \
  --text_encoder /workspace/models/text_encoders/split_files/text_encoders/qwen_2.5_vl_7b.safetensors \
  --dataset_config /workspace/musubi-tuner/dataset/dataset.toml \
  --edit \
  --control_image_path workspace/dataset_KONTEXT/samples/test.png \
  --prompt "Insert a thin grey low opacity bodywriting" \
  --resize_control_to_official_size \
  --mixed_precision bf16 \
  --weighting_scheme none \
  --optimizer_type adamw \
  --learning_rate 5e-5 \
  --gradient_accumulation_steps 1 \
  --max_data_loader_n_workers 2 \
  --network_dim 16 \
  --network_alpha 16 \
  --timestep_sampling shift \
  --discrete_flow_shift 2.2 \
  --max_train_epochs 100 \
  --save_every_n_epochs 10 \
  --seed 5 \
  --optimizer_args weight_decay=0.00001 \
  --max_grad_norm 0 \
  --lr_scheduler constant_with_warmup \
  --lr_warmup_steps 60 \
  --xformers \
  --output_dir /workspace/musubi-tuner/output \
  --output_name Bodywriting_Qwen_Musubi
